### DISCUSSION TITLE
Ambassador Program

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#12] [General]
### What is Darwinia  Ambassador Program?
1. It is a project to allocate the current resources appropriately in the DCDAO.
2. All the community members can apply for this program.
3. The distribution of rewards is result-oriented. 
4. You have to submit your work report in dework by the **3rd day** of the following month. Any late submission will not be accepted. Click [here](https://docs.google.com/document/d/1qWeVjT1Y7I5JA8zZub7Exl_2M-mIRg_4RRy4Iz0XV0Y/edit) to learn the template.
5. The rewards will be sent before the 10th of every month.
### What will you do?
As an ambassador, the first thing is to understand the DCDAO vision. Learn more [details](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x257aff1540719475411091706f7b200507fe9b9fe7397a8a4347fbc9d7234cb6) about it. There are three key points as the following:
1. Enrich and empower  the Darwinia ecosystem
2. Innovate and experiment with great ideas 
3. Leverage current key resources

So before you take action, you have to what kind of contribution you can make to the community based on these elements. 

You can take a reference of the community manager’s  OKR. Click [here](https://www.notion.so/dcdao/May-OKR-Community-Manager-228114d9c2694c8f9cc092b09d3d66e5).
### What will you get?
Darwinia treasury funding for Ambassador Program is about 2000-3000 USD per month. The breakdown of the budget is as follows:
<img width="315" alt="image" src="https://github.com/dcdao/discussion/assets/109792098/bc10a1e4-5ab2-4e5d-8242-90c0473734c9">

###  How to apply?
1. Fill out the Google form first to leave your basic information. Click [here](https://docs.google.com/forms/d/e/1FAIpQLSemwTC2shQ6HNCDja4UPe-CRGU2u5NFXYF-3u4j2Z9bbWidsA/viewform?usp=send_form) to apply!
2. The community manager will review the application and select the candidates to interview.

**Comment by 2xJDubs** [comment]
Will the KPIs for Twitter be the same as it is now?  

Also there should be some definition of what differentiates a Senior Ambassador from an Ambassador.

**Comment by hackfisher** [comment]
Thanks for the intro from Cindy.

I don't think current KPI is a good way for ambassador, as ambassador come here to the core community should be they are interested in it and attracted by the project's vision, they can contribute be helping the community through communication and education, that it is important.

And the reward should only be the basic for support, most attractive for the ambassadors should be the opportunities to talk to core contributors/directly, access to the global network of darwinia and more, we could find the right ambassadors for the projects in this way, it is *NOT* only a part time offer/job.

This proposal looks rough and inadequate, I'm looking for the improved one which could be better.

**Reply by 2xJDubs** [reply]
I agree with all of this.  The current Dework setup can bring in people that are just looking for money and not interested really in Darwinia.  It is hard to find those right people that are truly engaged to see the project succeed while helping the community.  When I started as an Ambassador there was no pay for us and it was more about helping the community along with access to the team to understand what was happening better.

**Comment by hackfisher** [comment]
And I against using KPI, use OKR instead.

**Reply by SasoLithops** [reply]
Hi!

If I may add although just supporter now, when I was part of community management I missed direct communication to team members. Would be good if community managers could have a Channel where they can reach out to team and not wait a couple of days for reply. I think this is already improving in Discord.
Also like Jesse said there were many engaging just for money not for love of project,not educating themselves in what Darwinia is and does. You have to take that in consideration. Sharring a tweet does not equal community management. I think like you propose supporters and community managers should be well distinguished. 
Also remember Laki and Jesse were here daily through whole bear market. Those are the most valuable ambassadors.
I would also keep rewards for video tutorials for users like Dustin or Greyhood were doing. I do wish we get some member good with video editing. It really helps users to follow a video tutorial.

**Reply by Cindy884** [reply]
Thks for your reply Saso.
1. I will host the Darwinia Ambassador meeting monthly in the future. Welcome to join us when you are free.
2. Yeah, you pointed out an important part. And I think it is a key challenge for the community manager to allocate resources. 
Hope to hear more voices from the community.

**Comment by hackfisher** [comment]
This link can not be viewed @Cindy884 : 

https://www.notion.so/dcdao/May-OKR-Community-Manager-115869bc4c264913a985633e67e0cbc9

**Reply by Cindy884** [reply]
I think you can view it now.:)

**Comment by hackfisher** [comment]
> This proposal looks rough and inadequate, I'm looking for the improved one which could be better.

@Cindy884 I found that the content has been updated and the latest changes LGTM.
